VERSION

```diff
@@ -1 +1 @@
-1.1.4
+1.1.5
```

Comment:
I've been meaning to ask, why wasn't this 1.1.4.90? (Or something similar) How do we distinguish between the `1.1` branch and the `1.1.4` release?

Reply:
With the git SHA. Is it really a problem? The release process is already fairly complex and it prevents us from doing more frequent releases.